### PR TITLE
Added 'Thin device space monitoring' feature page.

### PR DESCRIPTION
**Pull request by akashihi**
Repeating https://github.com/oVirt/ovirt-site/pull/1509

### CODE REVIEW

**Comment by rollandf** [comment]
I think that the errors are because we are missing the needed headers.

**Comment by gregsheremeta** [comment]
> I think that the errors are because we are missing the needed headers.

Hm, could be. I wish it would actually say that :/

@akashihi can you try adding this to the top:

```
---
title: Thin device space monitoring
authors: dchaplyg
feature_name: Thin device space monitoring
feature_modules: storage
feature_status: work in progress
---

```

**Comment by akashihi** [comment]
@gregsheremeta added headers and fixed typos in text

File: source/develop/release-management/features/storage/thin-device-monitoring.html.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+*   Last updated: July 31, 2018
+
+
+# Detailed description
+
+Storage domain can be deployed on top of thin provisioned storage, like LVM thin pool or VDO deduplicated/compressed device. In that case oVirt will report 'virtual' size of a storage domain, not taking into account its underlying limitations.
+
+For example, storage domain of 50G size can be built on top of thin pool with size of 50G, shared with two other logical volumes. Thin pool may have only 20G of actual free space and, because of that, oVirt user will get a 'out of space' error after writing 20G to a empty 50G partition. 
+
+As a solution of that problem, i propose to add a third monitoring parameter to the storage domain, a 'minimum guaranteed free size' or 'physical size' monitor and report that parameter.
```
Comment (rollandf) [inline source/develop/release-management/features/storage/thin-device-monitoring.html.md]: i propose -> I propose

File: source/develop/release-management/features/storage/thin-device-monitoring.html.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+# Implementation details
+
+While reporting physical size seem to be simple and obvious, implementing that feature could be challenging. To make it less challenging we will limit the problem to hyperconverged deployment - storage domain on gluster volumes, made from a local storage. In that case oVirt will have a direct access to all the storage details using local VDSM.
+
+But even in that case local storage data will not be enough. Gluster volume size/usage are reported per volume and are same on every gluster or vdsm host. On the other hand thin storage statistic is  reported per block device for each brick and may differ for each gluster host. Therefore we need a way to match block device to brick and brick to volume. Even worse - we may have a stack of thin devices (thin lvm on top of deduplicated device) and have to correctly resolve those dependencies.
```
Comment (rollandf) [inline source/develop/release-management/features/storage/thin-device-monitoring.html.md]: in that case,

**Comment by gregsheremeta** [comment]
[ rebased ]

**Comment by sandrobonazzola** [comment]
still relevant?

**Comment by akashihi** [comment]
Well, the feature is merged, probably nice to have it documented

**Comment by sandrobonazzola** [comment]
@rollandf should we close this PR due to inactivity?

**Comment by sandrobonazzola** [comment]
Closing due to unresponsiveness

**Pull request closed without merging**